apps/oc_chef_authz/src/oc_chef_authz_db.erl

```diff
@@ -148,6 +148,19 @@ statements(pgsql) ->
       <<"UPDATE groups SET last_updated_by= $1, updated_at= $2, name= $3"
         "WHERE id= $4">>},
      {delete_group_by_id, <<"DELETE FROM groups WHERE id= $1">>},
+     {list_policies_for_org, <<"SELECT name, policy_group FROM policies WHERE org_id= $1">>},
```

Comment:
This was added to give us a starting point in the cts. It is interesting as name isn't sufficient to generate a url. This is the only list query i could find that returns two items. With regard to order by, that would be to support manage and the ux is still to be determined.